pkgs/development/python-modules/configargparse/default.nix

```diff
@@ -7,7 +7,7 @@
 
 buildPythonPackage rec {
   pname = "ConfigArgParse";
-  version = "1.3";
+  version = "1.4";
 
   src = fetchFromGitHub {
     owner = "bw2";
```

Comment:
You forgot to update the hash.

Reply:
Thanks. Fixed.

The package now follows PEP 0503.